docs/docker-example/Dockerfile

```diff
@@ -24,6 +24,8 @@ RUN git clone https://github.com/onnx/tutorials.git
 # Install clang
 RUN apt-get install -y lsb-release wget software-properties-common
 RUN bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)"
+# For development
```

Comment:
You need to add these two lines in the Docker.md file as well. We have a tool that make sure that the documentation and scripts are in syncs. This is illustrated by this in the doc

```
[same-as-file]: <> (docs/docker-example/Dockerfile)
```

You can simply copy and paste the entire Dockerfile in the docker.md file. Running `make check-docs` will verify that all is good if you want to test that it is fine before restarting the CIs